docs/METRICS.md

```diff
@@ -0,0 +1,108 @@
+
+# Snap Static and Dynamic Metrics
+
+Snap Framework supports two types of metrics. They are static and dynamic metrics. A Snap metric consists of a namespace and value pair.
+
+### Static Metrics
+A namespace having no wildcard and only string literals separated by slashes is a static metric. For example:
+```
+/intel/cassandra/node/zeus/type/Cache/scope/KeyCache/name/Requests/OneMinuteRate
+/intel/cassandra/node/zeus/type/ClientRequest/scope/CASRead/name/Unavailables/FiveMinuteRate
+/intel/cassandra/node/apollo/type/Cache/scope/RowCache/name/Hits/OneMinuteRate
+/intel/cassandra/node/apollo/type/ClientRequest/scope/RangeSlice/name/Latency/OneMinuteRate
```

Comment:
I think it should hopefully be consistent inside a plugin. But it is not something we can control.